vagrant_variables.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+# Variables for in-repo testing with VirtualBox
+#
+
+# node parameters
+
+vm_memory: 1024
```

Comment:
Only a gig of mem?  What's going to be running on these nodes?

Reply:
Just ansible to configure ssh...

Reply:
After slack conversation, I have a better understanding of the usage of this node.